content-security-policy/support/resource.py

```diff
@@ -1,5 +1,5 @@
 def main(request, response):
     headers = []
-    headers.append(("Access-Control-Allow-Origin", "*"))
+    headers.append((b"Access-Control-Allow-Origin", b"*"))
 
-    return headers, "{ \"result\": \"success\" }"
+    return headers, u"{ \"result\": \"success\" }"
```

Comment:
```suggestion
    return headers, b"{ \"result\": \"success\" }"
```

Another of the same. Not sure if we should default to str or bytes.